types/luxon/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for luxon 2.4
+// Type definitions for luxon 3.0
```

Comment:
Function was added in 3.0.1 not 3.0.0
```suggestion
// Type definitions for luxon 3.0.1
```